guides/source/configuring.md

```diff
@@ -75,6 +75,7 @@ Below are the default values associated with each target version. In cases of co
 - [`config.add_autoload_paths_to_load_path`](#config-add-autoload-paths-to-load-path): `false`
 - [`config.log_file_size`](#config-log-file-size): `100 * 1024 * 1024`
 - [`config.precompile_filter_parameters`](#config-precompile-filter-parameters): `true`
+- [`config.active_record.encryption.hash_digest_class`](#config-active-encryption-hash-digest-class): `OpenSSL::Digest::SHA256`
```

Comment:
Can you move this up after `config.active_record.allow_deprecated_singular_association_name` to keep the list alphabetical?

(I'm now noticing the `belongs_to_required...` config is out of order as well, I'll fix that)